editor.planx.uk/src/routes/views/draft.tsx

```diff
@@ -69,12 +69,18 @@ const fetchSettingsForDraftView = async (
                 favicon
               }
               name
-              settings
+              teamSettings: team_settings {
```

Comment:
```suggestion
              settings: team_settings {
```

nit: I think we maintain this as just `settings`

This means we can just use `team.settings` instead of the more verbose and less meaningful `team.teamSettings`.